unstructured/utils.py

```diff
@@ -318,7 +330,7 @@ def scarf_analytics():
         pass
 
 
-def ngrams(s: str, n: int) -> List:
+def ngrams(s: List[str], n: int) -> List[Tuple[str]]:
```

Comment:
Now that we're past 3.7, add a `from __future__ import annotations` import at the top of the module and use lowercase `list`, `dict`, and `tuple` for typing.